### PR TITLE
dietary requirements field added

### PR DESCRIPTION
**Pull request by umxr**

### CODE REVIEW

File: src/views/create-order/CreateOrderView.js [hunk 1]
```diff
@@ -134,247 +137,302 @@ const CreateOrderView = (props: Props) => {
...
               />
               <Form.Control.Feedback type="invalid">
-                We need your phone number is required, what if we can't locate you?
+                We need your phone number is required, what if we can't locate
+                you?
               </Form.Control.Feedback>
-              <br/>
-
+              <br />
+              <Form.Label>Dietary Requirements</Form.Label>
```
Comment (mahfuzhannan) [inline src/views/create-order/CreateOrderView.js:345]: Is this only applicable to delivery only? Think we should add it to all orders maybe?

Comment (umxr) [reply src/views/create-order/CreateOrderView.js:345]: If you look at line 246. Ive added it there aswell. Should it does apply to all orders

Comment (mahfuzhannan) [reply src/views/create-order/CreateOrderView.js:345]: Ahh I see, but I think we should add it under the "Iftar Packs" section, what you reckon?

Comment (umxr) [reply src/views/create-order/CreateOrderView.js:345]: sure, that would work, would reduce duplication aswell. Ill make that change now

**Pull request merged**